Client/Frontend/Browser/TabManager.swift

```diff
@@ -59,8 +59,8 @@ class TabManager: NSObject {
     }
 
     fileprivate(set) var tabs = [Tab]()
-    fileprivate var previousTab: Tab?
```

Comment:
Key part of this patch was to remove the added state management of this variable. (Note also this was non-weak).